config/environments/development.rb

```diff
@@ -52,5 +52,5 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.file_watcher = ActiveSupport::FileUpdateChecker
```

Comment:
Does this fix your server issue?  We probably don't want this change in the repo since FileUpdateChecker will continuously poll for changes on the files, which can be super CPU slow, but if this fixes the issue on your server we should be able to fix it another way that doesn't require falling back to file polling.